### PR TITLE
merge strings for train 32993

### PR DESCRIPTION
**Pull request by bug-mirror**

### CODE REVIEW

File: locale/templates/main.ftl [hunk 1]
```diff
@@ -344,6 +344,17 @@ plan-details-header = Product details
 plan-details-show-button = Show details
 plan-details-hide-button = Hide details
 plan-details-total-label = Total
+plan-details-list-price = List Price
+
+## coupons
```
Comment (peiying2) [inline locale/templates/main.ftl:349]: If this is a section comment, it needs to have an extra line break between this and the strings below. 

See documentation on this; https://projectfluent.org/fluent/guide/comments.html

File: locale/templates/auth.ftl [hunk 2]
```diff
@@ -249,6 +288,34 @@ subscriptionAccountDeletion-title = Sorry to see you go
...
+subscriptionAccountReminderFirst-content-select = Select “Create Password” to set up a new password and finish verifying your account.
+subscriptionAccountReminderFirst-action = Create Password
+subscriptionAccountReminderFirst-action-plaintext = { subscriptionAccountReminderFirst-action }:
+
+subscriptionAccountReminderSecond-subject = Final reminder: Setup your account
+subscriptionAccountReminderSecond-title = Welcome to { -brand-firefox }!
+subscriptionAccountReminderSecond-content-info = A few days ago you created a { -product-firefox-account } but never verified it. We hope you’ll finish setting up your account, so you can use your new subscription.
+subscriptionAccountReminderSecond-content-select = Select “Create Password” to set up a new password and finish verifying your account.
+subscriptionAccountReminderSecond-action = Create Password
+subscriptionAccountReminderSecond-action-plaintext = { subscriptionAccountReminderSecond-action }:
```
Comment (peiying2) [inline locale/templates/auth.ftl:317]: Do we need to expose this string?

File: locale/templates/auth.ftl [hunk 3]
```diff
@@ -249,6 +288,34 @@ subscriptionAccountDeletion-title = Sorry to see you go
...
+#  $productName (String) - The name of the subscribed product, e.g. Mozilla VPN
+subscriptionAccountFinishSetup-content-create = Next, you’ll create a Firefox account password and download { $productName }.
+subscriptionAccountFinishSetup-action = Create a password
+
+subscriptionAccountReminderFirst-subject = Reminder: Finish setting up your account
+subscriptionAccountReminderFirst-title = You can’t access your subscription yet
+subscriptionAccountReminderFirst-content-info = A few days ago you created a { -product-firefox-account } but never verified it. We hope you’ll finish setting up your account, so you can use your new subscription.
+subscriptionAccountReminderFirst-content-select = Select “Create Password” to set up a new password and finish verifying your account.
+subscriptionAccountReminderFirst-action = Create Password
+subscriptionAccountReminderFirst-action-plaintext = { subscriptionAccountReminderFirst-action }:
```
Comment (peiying2) [inline locale/templates/auth.ftl:310]: Does this need to be exposed?

File: locale/templates/auth.ftl [hunk 4]
```diff
@@ -46,6 +47,10 @@ subplat-cancel-plaintext = { subplat-cancel }:
 subplat-reactivate = Reactivate subscription
 subplat-reactivate-plaintext = { subplat-reactivate }:
 subplat-update-billing = Update billing information
+subplat-privacy-policy = { -brand-mozilla } Privacy Policy
+subplat-privacy-policy-plaintext = { subplat-privacy-policy }:
```
Comment (peiying2) [inline locale/templates/auth.ftl:51]: Do we need to expose this string to l10n?

File: locale/templates/auth.ftl [hunk 5]
```diff
@@ -46,6 +47,10 @@ subplat-cancel-plaintext = { subplat-cancel }:
...
+subplat-cloud-terms = { -product-firefox-cloud } Terms of Service
+subplat-cloud-terms-plaintext = { subplat-cloud-terms }:
```
Comment (peiying2) [inline locale/templates/auth.ftl:53]: Same here?

**Review by peiying2 — CHANGES_REQUESTED** [review]
One change request in the main.ftl file. 

Question for strings that contain no localizable content such as this one ` subscriptionAccountReminderFirst-action-plaintext = { subscriptionAccountReminderFirst-action }: `

File: locale/templates/auth.ftl (hunk 4 above)
Comment (clouserw) [reply locale/templates/auth.ftl:51]: @xlisachan @LZoog

File: locale/templates/auth.ftl (hunk 5 above)
Comment (clouserw) [reply locale/templates/auth.ftl:53]: @xlisachan @LZoog

File: locale/templates/main.ftl (hunk 1 above)
Comment (clouserw) [reply locale/templates/main.ftl:349]: @IvoJP

**Comment by clouserw** [comment]
Thanks for the review.  Most folks are out for the holiday weekend so we'll probably have to fix these and re-extract next week.

**Comment by peiying2** [comment]
> Thanks for the review. Most folks are out for the holiday weekend so we'll probably have to fix these and re-extract next week.

Sounds good. I will be off next week. Feel free to merge it afterwards

File: locale/templates/auth.ftl (hunk 4 above)
Comment (LZoog) [reply locale/templates/auth.ftl:51]: These strings expose the same string as the non-plaintext version (and we want them to always be identical, which is why we're using a peaceable) but they also have a colon, so we do need them. This is the same for the other comments.

File: locale/templates/main.ftl (hunk 1 above)
Comment (IvoJP) [reply locale/templates/main.ftl:349]: I'm not sure how to handle this request - there isnt an extra line for groups anywhere else in that file so adding it for this section would be inconsistent. I also couldn't find anything in the documentation that recommended/required it (i could be wrong) aside from the example employing the change requested here.

File: locale/templates/auth.ftl (hunk 4 above)
Comment (peiying2) [reply locale/templates/auth.ftl:51]: @LZoog you are right, it contains  a`:` in addition to the brand name, so they should stay.

**Pull request closed without merging**